package.json

```diff
@@ -125,6 +127,7 @@
     "hammer-simulator": "0.0.1",
     "hugo-bin": "^0.82.2",
     "ip": "^1.1.5",
+    "jasmine": "^4.0.2",
```

Comment:
How about using the latest jasmine 3.x so that we don't use duplicate packages? We can update to 4.x when karma-jasmine is updated too.

Reply:
Damn, I'm very confused as to which packages get pulled now. I had originally found a 3.x (3.6 I think), then this morning I looked in the node-modules before adding Jasmine and `jasmine-core` was 4.0.1. I'll double check on the main branch after clearing node-modules 😄

Reply:
Oh, so on main, there's two `jasmine-core` pulled:
- one inside `karma-jasmine`, which is 3.99.1 (brought from it requiring `^3.6.0`)
- one at the root of node-modules, which is `4.0.1` (this one I'm not sure of the origin, but I'd guess `karma-jasmine-html-reporter` with `>=3.8`)

I'll try and align if I can 😄

Reply:
You should stick to 3.x which is what we specify. The v4.x is a dependency of a dependency, not a direct one.